doc/controller_configuration/controller_configuration_tutorial.rst

```diff
@@ -30,146 +79,407 @@ The ``controllers.yaml`` configuration file is located in the ``robot_moveit_con
       - panda_finger_joint1
       - panda_finger_joint2
 
-There are many different parameters that can be used for different types of controllers.
+The mapping from ROS controller ``name`` to a ``MoveItControllerHandle`` ``type`` is done by using the pre-defined *integration types* ``FollowJointTrajectory`` and ``GripperCommand``, which can interface any controller implementing the corresponding action interface (i.e. not only `ROS Control <http://wiki.ros.org/ros_control>`_ controllers).
+
+The ``action_ns`` setting specifies the *action server* topic exposed by the ROS controller. The full topic name is ``<name>/<action_ns>``.
+If you were to list topics by using ``rostopic list`` with the above two ROS controllers loaded, you would see something like the following: ::
+
+/panda_arm_controller/follow_joint_trajectory/goal
+/panda_arm_controller/follow_joint_trajectory/feedback
+/panda_arm_controller/follow_joint_trajectory/result
+/gripper_controller/gripper_action/goal
+/gripper_controller/gripper_action/feedback
+/gripper_controller/gripper_action/result
+
+There are many different parameters that can be defined for the two types of simple controller interfaces.
 
 FollowJointTrajectory Controller Interface
-------------------------------------------
-The parameters are:
- * *name*: The name of the controller.  (See debugging information below for important notes).
- * *action_ns*: The action namespace for the controller. (See debugging information below for important notes).
- * *type*: The type of action being used (here FollowJointTrajectory).
- * *default*: The default controller is the primary controller chosen by MoveIt for communicating with a particular set of joints.
- * *joints*: Names of all the joints that are being addressed by this interface.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * ``name``: The name of the controller.  (See debugging information below for important notes).
+ * ``action_ns``: The action namespace for the controller. (See debugging information below for important notes).
```

Comment:
It seems the "debugging info" is now `above`?

Also it would be super helpful to add more debugging tips. Currently my robot is not moving and I have no idea why. Is there a way to make `ros_control` acknowledge that it received a trajectory? Print the content? This kind of thing

Reply:
Yes, this was moved too many times. We can get this fixed.